cvs -z3 diff textctrl.cpp (in directory E:\Dev\wxWindows\src\msw\)
Index: textctrl.cpp
===================================================================
RCS file: /home/wxcvs/wxWindows/src/msw/textctrl.cpp,v
retrieving revision 1.100
diff -r1.100 textctrl.cpp
71a72,93
> // cygwin does not have these defined for richedit
> #ifndef ENM_LINK
>     #define ENM_LINK				0x04000000
> #endif
> 
> #ifndef EM_AUTOURLDETECT
>     #define EM_AUTOURLDETECT		(WM_USER + 91)
> #endif
> 
> #ifndef EN_LINK
>     #define EN_LINK					0x070b
> 
>     typedef struct _enlink
>     {
>         NMHDR nmhdr;
>         UINT msg;
>         WPARAM wParam;
>         LPARAM lParam;
>         CHARRANGE chrg;
>     } ENLINK;
> #endif
> 


 	  	 
